frontend/src/pages/RecipientRecord/pages/MergeGoals/components/__tests__/FinalGoalCard.js

```diff
@@ -175,4 +175,34 @@ describe('FinalGoalCard', () => {
     const label = document.querySelector('.ttahub-final-goal--status .ttahub-final-goal--status-label');
     expect(label.textContent).toBe('Not started');
   });
+  it('shows the correct status for draft + not started', async () => {
+    act(() => renderTest([
+      {
+        ...baseGoals[0],
+        goalStatus: 'Not Started',
+      },
+      {
+        ...baseGoals[1],
+        goalStatus: 'Draft',
+      },
+    ]));
+
+    const label = document.querySelector('.ttahub-final-goal--status .ttahub-final-goal--status-label');
+    expect(label.textContent).toBe('Not started');
```

Comment:
Should we also be seeing a 'Draft' here?

Reply:
No, not started + draft = not started